src/vhdlConvertor/packageHeaderParser.cpp

```diff
@@ -56,6 +79,7 @@ void VhdlPackageHeaderParser::visitPackage_declarative_part(
 
 void VhdlPackageHeaderParser::visitPackage_declarative_item(
 		vhdlParser::Package_declarative_itemContext *ctx) {
+	// VHDL-93 syntax, maybe -- 
```

Comment:
The syntax shown here doesn't match the VHDL-2008 syntax.  I'm not sure where it came from.